From 32071361ac6eba4b89da55370e05d603a1db5a99 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 09:52:32 +0800
Subject: [PATCH] Test case for included FrontPage.h changing - String.c file

Description: Change the FrontPage.h files which included in the MdeModulePkg\Application\UiApp\String.c.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/FrontPage.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Application/UiApp/FrontPage.h b/MdeModulePkg/Application/UiApp/FrontPage.h
index 36207287b9..871d78e970 100644
--- a/MdeModulePkg/Application/UiApp/FrontPage.h
+++ b/MdeModulePkg/Application/UiApp/FrontPage.h
@@ -32,7 +32,7 @@ extern EFI_FORM_BROWSER2_PROTOCOL      *gFormBrowser2;
 //
 // These are defined as the same with vfr file
 //
-#define FRONT_PAGE_FORM_ID             0x1000
+#define FRONT_PAGE_FORM_ID             0x1010
 
 #define LABEL_FRANTPAGE_INFORMATION    0x1000
 #define LABEL_END                      0xffff
-- 
2.27.0.windows.1

